### ISSUE TITLE
Speed calculation is not objective

### ISSUE THREAD

**Issue by Iprogrammerr** [zerocracy/farm#1895]
When programmers speed is calculated, the duration of the code reviews is taken into account. It is not objective measure, as it is often the case that reviewers don't look at assigned to them PR's for a few days. Moreover, it is not rare for them to report new issues hours or days after first review, which creates second, not dependent on PR author skills time overhead. Therefore , it should be changed to take this into account, if it ought to be objective, as it is claimed to be.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
@iprogrammerr[/z](https://www.0crat.com/u/iprogrammerr) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by g4s8** [comment]
@Iprogrammerr the performer is responsible for closing an issue as soon as possible, see these links: https://www.zerocracy.com/policy.html#7 and https://www.yegor256.com/2014/04/15/definition-of-done.html
Even if performer is waiting for review he/she is also responsible for it.
Also, there is no direct relation between issue and PR - an issue may have PR or may not have, or may have multiple PRs or one PR may close multiple issues, so 0crat will not link PRs to job tickets.

0crat should know that particular metric - the duration from start of order (when performer was assigned) to end of order (when ticket was closed), this speed metric is needed for 0crat internal calculations first of all, not for the user, 0crat should not care why ticket was closed later, because of PR review or something else, 0crat should now only time interval from start to end.

**Comment by 0crat** [comment]
Job `gh:zerocracy/farm#1895` is not assigned, can't get performer

**Comment by 0crat** [comment]
This job is not in scope
